### ISSUE TITLE
Air Tap not working on HoloLens1 remoting/device, but works in editor

### ISSUE THREAD

**Issue by evhan55** [microsoft/MixedRealityToolkit-Unity#6413]
## Description

In my Unity/MRTK app, I can get the 'air tap' gesture to work in the editor by using the space bar + click to simulate the gesture, but trying an actual air tap in Holographic Remoting and in the HoloLens 1 itself does not work.

## To reproduce

*Working Scenario*
1. Click 'play' on my app, in the editor
2. Press 'space bar' and click to simulate an air tap
3. App responds

*Buggy Scenario 1*
1. Go to Window > Holographic Emulation
2. Connect to my HoloLens 1 (by setting Emulation Mode to 'Remote to Device')
3. Click 'play' on my app
4. Do air tap gesture
5. App does not respond

*Buggy Scenario 2*
1. Build app in Unity
2. Use Visual Studio 2019 to run app on HoloLens 1
3. Do air tap gesture
4. App does not respond

## Expected behavior

App should respond to 'air tap' gesture in Holographic Emulation mode and through Unity, on device itself.

## Setup

- Unity Version [2018.4.12f1]
- MRTK Version [v2.0.0]

## Target Platform

- HoloLens 1

## Additional context

I've tried setting the 'Input Simulation Service' 'Hand Simulation Mode' to 'Gestures' vs. 'Articulated', and this did not fix it.

Seems to be related to these closed issues:
https://github.com/microsoft/MixedRealityToolkit-Unity/issues/3644
https://github.com/microsoft/MixedRealityToolkit-Unity/issues/3909

**Comment by keveleigh** [comment]
In Unity, under Window -> Package Manager, which version of the Windows Mixed Reality package do you have?

**Comment by evhan55** [comment]
@keveleigh

It says Version 1.0.14

**Comment by evhan55** [comment]
Hello! Just wanted to report I am still seeing this in MRTK 2.2 as well.

**Comment by keveleigh** [comment]
I haven't been able to repro this recently. @evhan55, are you still seeing it?

**Comment by evhan55** [comment]
Hi @keveleigh,

I ended up getting help on this StackOverflow issue: https://stackoverflow.com/questions/59502650/why-is-air-tap-gesture-not-working-on-hololens1-in-my-unity-mrtk-app/59655477?noredirect=1#comment105537134_59655477

Turns out I had to actually write code to handle the air-taps from MRTK and send them over correctly to my Unity app. The reason I thought it was working in the emulator was because the 'air-tap' in the emulator actually also registered a 'click' from Unity, which was being handled by my app just fine.

I have a new issue altogether, with my app finally working in the Holographic emulation (air-taps and all), but then it doesn't work when deployed to my HL1 device. There seems to be an offset in the coordinate systems or something, but I'm not sure, should I open a new issue for that?

This issue is considered resolved for me by the StackOverflow solution linked above, thank you!
